ccm_web/client/src/components/CustomTable.tsx

```diff
@@ -78,10 +78,6 @@ function CustomTable<T extends TableEntity> (props: TableProps<T>): JSX.Element
               count={tableRows.length}
               rowsPerPage={rowsPerPage}
               page={page}
-              SelectProps={{
-                inputProps: { 'aria-label': 'rows per page' },
-                native: true
-              }}
```

Comment:
I see this was deprecated in MUI. was there a replacement needed? or is deleting this did not had any side effects?

Reply:
I don't think this snippet was actually having any effect, since these props are supposed to be used within the tablePagination footer. I think its possible that it must have been from a previous version or copied from a template that had an option to change the number of rows per page (hence the use of inputProps). After removing this, the aria-labels look the same when I compare to production.